ee/tabby-ui/components/ui/icons.tsx

```diff
@@ -204,6 +206,9 @@ function IconChevronRight({
     </svg>
   )
 }
+
+// FIXME: Remove to use ChevronLeft from lucide-react
+// currently ui/calendar is using ChevronLeft as the default option.
```

Comment:
```suggestion

// FIXME(wwayne): Review each icons and consider re-export from `lucide-react`.
```

Move this fix me to top of the file.